docs/fundamentals/as-avs.txt

```diff
@@ -0,0 +1,262 @@
+.. _laravel-atlas-search-avs:
+
+============================
+Atlas Search & Vector Search
+============================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: code example, semantic, embeddings
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to perform searches on your documents
+by using the Atlas Search and Vector Search features. {+odm-long+}
+provides an API to use these features directly with your models.
+
+To learn more about these features, see the following pages in the Atlas
+documentation:
+
+- :atlas:`Atlas Search Overview </atlas-search/atlas-search-overview/>`
+- :atlas:`Atlas Vector Search Overview </atlas-vector-search/vector-search-overview/>`
+
+.. note:: Deployment Compatibility
+
+   You can use the Atlas Search and Vector Search features only when
+   connecting to MongoDB Atlas clusters. These features are not available for
+   self-managed deployments.
+
+This page describes how to create the appropriate indexes for each type
+of search and provides examples of how to use the {+odm-short+} to
+perform searches.
+
+Atlas Search
+------------
+
+In this section, you can learn how to create Atlas Search indexes
+and perform searches in the {+odm-short+}. The Atlas Search API
+internally uses the ``$search`` aggregation operator to perform queries.
+To learn more about this operator, see the :atlas:`$search
+</atlas-search/aggregation-stages/search/>` reference in the Atlas documentation.
+
+.. _laravel-as-index:
+
+Create an Atlas Search Index
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. TODO in DOCSP-46230
+
+Perform Queries
+~~~~~~~~~~~~~~~
+
+In this section, you can learn how to use the Atlas Search API in the
+{+odm-short+}.
+
+General Queries
+```````````````
+
+The {+odm-short+} provides the ``search()`` method as a query
+builder method and as an Eloquent model method. You can use the
+``search()`` method to run Atlas Search queries on documents in your
+collections.
+
+You must pass an ``operator`` parameter to the ``search()`` method that
+is an instance of ``SearchOperatorInterface`` or an array. You can
+create an instance of ``SearchOperatorInterface`` by calling the
+``Search::text()`` method and passing the field you are
+querying and your search term or phrase.
+
+You must include the following import statement in your application to
+create a ``SearchOperatorInterface`` instance:
+
+.. code-block:: php
+
+   use MongoDB\Builder\Search;
+
+The following code uses the index created in the preceding
```

Comment:
```suggestion
The following code uses the ``title`` index created in the preceding
```
S: PLace the name of the index here

Reply:
Since the index name isn't included in the code example anyways, I'll just rephrase